dot/parachain/candidate-validation/candidate_validation.go

```diff
@@ -43,9 +43,19 @@ func NewCandidateValidation(overseerChan chan<- any) *CandidateValidation {
 }
 
 // Run starts the CandidateValidation subsystem
-func (cv *CandidateValidation) Run(context.Context, chan any, chan any) {
-	cv.wg.Add(1)
```

Comment:
I guess we then should also remove wg from CandidateValidaton? It is not used anymore